src/runner.cpp

```diff
@@ -7,8 +7,8 @@
 
 using namespace std;
 
-absl::StatusOr<vector<string>> run(Config c) {
-  string s = absl::StrJoin(c.getLines(), ",");
+absl::StatusOr<vector<string>> run(absl::StatusOr<Config> c) {
```

Comment:
`run` function should take `Config`. If the `Config` is invalid, it shouldn't be passed to `run` in the first place